modules/devShells.nix

```
@@ -17,6 +18,13 @@ mkTransposedPerSystemModule {
       An attribute set of packages to be used as shells.
       [`nix develop .#<name>`](https://nixos.org/manual/nix/stable/command-ref/new-cli/nix3-develop.html) will run `devShells.<name>`.
     '';
+    example = literalExpression ''
+    {
+      devShells.default = pkgs.mkShell {
+          nativeBuildInputs = with pkgs; [ wget bat git cargo ];
+        };
+    }
```

```suggestion
    # devShells =
    {
      default = pkgs.mkShell {
        nativeBuildInputs = with pkgs; [ wget bat cargo ];
      };
    }
```

Module docs don't repeat the path to the option.
No git.
Consistent indentation.